src/commonMain/kotlin/fr/acinq/lightning/NodeParams.kt

```diff
@@ -243,9 +245,12 @@ data class NodeParams(
                 maxAllowedFeeCredit = 0.msat
             )
         ),
-        minFinalCltvExpiryDelta = Bolt11Invoice.DEFAULT_MIN_FINAL_EXPIRY_DELTA,
+        // We use a long expiry delta here for a few reasons:
+        //  - we want to ensure we're able to get HTLC-success txs confirmed if our peer ignores our preimage
+        //  - we may be offline for a while, so we want our peer to be able to hold HTLCs and forward them when we come back online
+        minFinalCltvExpiryDelta = CltvExpiryDelta(144),
```

Comment:
Won't this very high value conflict with our own `paymentRecipientExpiryParams` on the sending side?